static/src/js/zustand/slices/__tests__/createProjectSlice.test.ts

```diff
@@ -32,13 +32,22 @@ jest.mock('../../../actions', () => ({
   addGranuleMetadata: jest.fn(),
   handleAlert: jest.fn(),
   handleError: jest.fn(),
-  setDataQualitySummaries: jest.fn(),
   toggleSpatialPolygonWarning: jest.fn(),
   updateCollectionMetadata: jest.fn()
 }))
 
 beforeEach(() => {
   MockDate.set(new Date('2025-01-01T00:00:00Z'))
+
+  const mockSetDataQualitySummaries = jest.fn((catalogItemId, summaries) => {
+    useEdscStore.setState((state) => {
+      state.dataQualitySummaries.byCollectionId[catalogItemId] = summaries
+    })
+  })
+
+  useEdscStore.setState((state) => {
+    state.dataQualitySummaries.setDataQualitySummaries = mockSetDataQualitySummaries
```

Comment:
```
state.dataQualitySummaries.setDataQualitySummaries = jest.fn()
```

You don't want to recreate the function within the mock like you did above, because if that needs to change in the future you don't want to keep two versions of the function in sync. And within this file you are now only using `toHaveBeenCalledWith`, so the mocked function doesn't need to actually do anything.